keyboards/mokey/mokey64/rules.mk

```diff
@@ -0,0 +1,23 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite      # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
```

Comment:
The photo of the PCB shows what looks like per-key LEDs; are they simple single color backlight LEDs or RGB LEDs? In the first case you should set `BACKLIGHT_ENABLE = yes` and configure `BACKLIGHT_PIN`; in the second case you can use RGBLIGHT as a temporary solution to enable them (however, at the moment you have `#define RGBLED_NUM 16`, which matches the typical underglow setup, not per-key), but ideally you should use `RGB_MATRIX_ENABLE = yes` for per-key RGB LEDs.